docs/tutorials/tabular/tabular-foundational-models.ipynb

```diff
@@ -0,0 +1,464 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# AutoGluon Tabular - Foundational Models\n",
+    "[![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/autogluon/autogluon/blob/master/docs/tutorials/tabular/tabular-foundational-models.ipynb)\n",
+    "[![Open In SageMaker Studio Lab](https://studiolab.sagemaker.aws/studiolab.svg)](https://studiolab.sagemaker.aws/import/github/autogluon/autogluon/blob/master/docs/tutorials/tabular/tabular-foundational-models.ipynb)\n",
+    "\n",
+    "In this tutorial, we introduce support for cutting-edge foundational tabular models that leverage pre-training and in-context learning to achieve state-of-the-art performance on tabular datasets. These models represent a significant advancement in automated machine learning for structured data.\n",
+    "\n",
+    "In this tutorial, we'll explore four foundational tabular models:\n",
+    "\n",
+    "1. **Mitra** - AutoGluon's new state-of-the-art tabular foundation model\n",
+    "2. **TabM** - Parameter-efficient ensembling for tabular deep learning\n",
+    "3. **TabICL** - In-context learning for large tabular datasets\n",
+    "4. **TabPFNv2** - Prior-fitted networks for accurate predictions on small data\n",
+    "\n",
+    "These models excel particularly on small to medium-sized datasets and can run in both zero-shot and fine-tuning modes."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Installation\n",
+    "\n",
+    "First, let's install AutoGluon with support for foundational models:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Individual model installations:\n",
+    "!pip install uv\n",
+    "!uv pip install autogluon.tabular[mitra]   # For Mitra\n",
+    "!uv pip install autogluon.tabular[tabm]     # For TabM\n",
```

Comment:
Note: TabM isn't a foundation model, it is just a really good deep learning model

Reply:
removed tabm from this tutorial and maybe we can add it to in-depth tutorial with realMLP